tabcmd/commands/datasources_and_workbooks/get_url_command.py

```diff
@@ -138,6 +141,24 @@ def get_view_url(url, logger):  # "views/wb-name/view-name" -> wb-name/sheets/vi
         view_name = GetUrl.get_name_without_possible_extension(view_name)
         return DatasourcesAndWorkbooks.get_view_url_from_names(workbook_name, view_name)
 
+    @staticmethod
+    def get_url_parts_from_custom_view_url(url, logger):
+        name_parts = url.split("/")  # ['views', 'wb-name', 'view-name', 'custom-view-id', 'custom-view-name']
```

Comment:
Could we do this custom view id logic once for Get_command and Export_command? Or perhaps at least move all the url parsing methods into datasources_and_workbooks so that we can keep an eye on the repeated code?

Reply:
sounds good, I've moved all the url parser logic to a separate file & class